src/views/teams/team-list.hbs

```diff
@@ -0,0 +1 @@
+Lorem ipsum dolor sit amet consectetur adipisicing elit. Eaque molestiae rem alias aut dicta numquam labore, molestias dolorum. Amet ut pariatur possimus. Ipsam ad nobis sint provident, architecto consequuntur omnis.
```

Comment:
1.- use this path `src/views/teams.hbs`
2.- Add an "empty" hbs like:
```
<div class="container-fluid">
    <div class="row">
        <nav id="sidebarMenu" class="col-md-3 col-lg-2 d-md-block bg-light sidebar collapse">
            <div class="sidebar-sticky pt-3">
                <!-- MENU-AT-RESOURCES -->
                {{> menu-at-resources }}
                <!-- MENU-AT-RESOURCES -->
            </div>
        </nav>

        <!-- TEAMS -->
        <main role="main" class="col-md-9 ml-sm-auto col-lg-10 px-md-5">
            <div
                class="d-flex justify-content-between flex-wrap flex-md-nowrap align-items-center pt-3 pb-2 mb-3 border-bottom">
                <h1 class="h2">TEAMS</h1>
            </div>
        </main>
        <!-- TEAMS -->
    </div>
</div>
```

Reply:
Got it